scipy/signal/ltisys.py

```diff
@@ -445,6 +423,13 @@ def impulse(self, X0=None, T=None, N=None):
         """
         return impulse(self, X0=X0, T=T, N=N)
 
+    def impulse2(self, X0=None, T=None, N=None, **kwargs):
```

Comment:
My understanding was that ```impulse2```/```step2``` are actually going to replace ```impulse``` and ```step```? I'm not sure though